include/exadg/postprocessor/error_calculation.cpp

```diff
@@ -134,6 +134,16 @@ ErrorCalculator<dim, Number>::setup(dealii::DoFHandler<dim> const &   dof_handle
     create_directories(error_data.directory, mpi_comm);
 }
 
+template<int dim, typename Number>
+void
+ErrorCalculator<dim, Number>::setup_after_coarsening_and_refinement(
```

Comment:
compare this to the `setup` just above these lines here.

Reply:
probably similar as for `OutputGenerator`